src/bind.js

```diff
@@ -18,8 +18,10 @@
  *
  * @return {Function}
  */
-function bind(callback) {
-  // write code here
+function bind(callback, ...argument) {
+  return function(...argument1) {
```

Comment:
write it in one line
```suggestion
  return (...numbers) => callback(...arg, ...numbers);
```

also, don't name `argument1 , 2, 3 ...`